samples/RoslynPadReplSample/MainWindow.xaml.cs

```diff
@@ -44,7 +44,12 @@ private void OnLoaded(object sender, RoutedEventArgs e)
             {
                 Assembly.Load("RoslynPad.Roslyn.Windows"),
                 Assembly.Load("RoslynPad.Editor.Windows")
-            });
+            }, RoslynHostReferences.NamespaceDefault.With(new[]
+            { 
+                MetadataReference.CreateFromFile(typeof(object).Assembly.Location),
+                MetadataReference.CreateFromFile(typeof(System.Text.RegularExpressions.Regex).Assembly.Location),
+                MetadataReference.CreateFromFile(typeof(System.Linq.Enumerable).Assembly.Location),
+            }));
```

Comment:
Other references are needed for snippet execution not to error out with errors like

```
error CS0234: The type or namespace name 'RegularExpressions' does not exist in the namespace 'System.Text' (are you missing an assembly reference?)
error CS0234: The type or namespace name 'Linq' does not exist in the namespace 'System' (are you missing an assembly reference?)
```